eslint.config.base.js

```diff
@@ -102,19 +102,6 @@ export function createConfig(opts = {}) {
           globals: { ...globals.browser },
         },
       },
-      withVitest && {
-        /**
-         * Configure Vitest globals on test files.
-         */
-        name: 'h5web/defaults/globals-vitest',
-        files: [
-          '**/__tests__/**/*.{js,jsx,ts,tsx}',
-          '**/*.test.{js,jsx,ts,tsx}',
-        ],
-        languageOptions: {
-          globals: { ...vitestPlugin.environments.env.globals },
-        },
-      },
```

Comment:
I realised we don't actually rely on Vitest's globals in our tests (e.g. `describe`, `it`, `beforeEach`, etc.) We always import those functions from the `vitest` package explicitly.